.husky/pre-commit

```diff
@@ -2,3 +2,4 @@
 source "$(dirname "$0")/_/husky.sh"
 
 yarn lint-staged
+yarn format
```

Comment:
No don't do that; configure this in lint-staged, `.lintstagedrc.yml` (it should already be configured)

Husky is generally for invoking lint-staged, but lint-staged is responsible for doing them

Reply:
If it was already there then why did manually running the formatter affect so many files? Shouldn't it have already been running automatically?

Reply:
Pre-commit only affects files you're committing; formatting manually will run on the files you specified, which is the entire project